backends/bmv2/common/annotations.h

```diff
@@ -20,9 +20,9 @@ limitations under the License.
 #include "frontends/p4/parseAnnotations.h"
 #include "ir/ir.h"
 
-namespace BMV2 {
+namespace P4C::BMV2 {
 
-using namespace P4::literals;
+using namespace ::P4C::P4::literals;
```

Comment:
@vlstill I think it would make sense to remove `P4` inner namespace here? So we can simply have "using namespace ::P4C::literals` or even `using namespace literals` if we're just in the `P4C` namespace. What do you think?

Reply:
I think this is worth discussing. Just wonder if it's better to keep this PR as it is to make the diffs more straightforward, and continue the discussion on further namespace refactorings in #4707?